static/app/components/events/interfaces/spans/utils.spec.tsx

```diff
@@ -1,27 +1,36 @@
 import {
   getCumulativeAlertLevelFromErrors,
-  getFormattedTimeRangeWithLeadingZero,
+  getFormattedTimeRangeWithLeadingAndTrailingZero,
 } from 'sentry/components/events/interfaces/spans/utils';
 
 describe('test utility functions', function () {
-  it('getFormattedTimeRangeWithLeadingZero', function () {
-    let result = getFormattedTimeRangeWithLeadingZero(
+  it('getFormattedTimeRangeWithLeadingAndTrailingZero', function () {
+    let result = getFormattedTimeRangeWithLeadingAndTrailingZero(
       1658925888.601534,
       1658925888.60193
     );
 
     expect(result.start).toEqual('1658925888.601534');
-    expect(result.end).toEqual('1658925888.060193');
+    expect(result.end).toEqual('1658925888.601930');
 
-    result = getFormattedTimeRangeWithLeadingZero(1658925888.601534, 165892588.060193);
+    result = getFormattedTimeRangeWithLeadingAndTrailingZero(
+      1658925888.601534,
+      165892588.060193
```

Comment:
```suggestion
      1658925888.060193
```
Expected a more recent date? You are missing 1 digit.

Reply:
oh nvm i see it's negative test.